tests/sentry/api/endpoints/test_source_map_debug.py

```diff
@@ -670,3 +670,143 @@ def test_sourcemap_in_file(self):
+                    "values": [
+                        {
+                            "type": "Error",
+                            "stacktrace": {
+                                "frames": [
+                                    {
+                                        "abs_path": "https://example.com/application.js",
+                                        "lineno": 1,
+                                        "colno": 39,
+                                    }
+                                ]
+                            },
+                        }
+                    ]
+                },
+            },
+            project_id=self.project.id,
+        )
+        resp = self.get_success_response(
+            self.organization.slug,
+            self.project.slug,
+            event.event_id,
+            frame_idx=0,
+            exception_idx=0,
+        )
+
+        error = resp.data["errors"][0]
+        assert error["type"] == "sdk_out_of_date"
+        assert error["message"] == "The SDK is out of date"
+        assert error["data"] == {
+            "sdkName": "sentry.javascript.browser",
+            "sdkVersion": "7.1.0",
+            "showMigrationGuide": False,
+        }
+
+    def test_raw_stacktrace_found(self):
+        event = self.store_event(
+            data={
+                "event_id": "a" * 32,
+                "release": "my-release",
+                "dist": "my-dist",
+                "sdk": {
+                    "name": "sentry.javascript.browser",
+                    "version": "7.1.0",
+                },
+                "exception": {
+                    "values": [
+                        {
+                            "type": "Error",
+                            "stacktrace": {
+                                "frames": [
+                                    {
+                                        "abs_path": "https://example.com/application.js",
+                                        "lineno": 1,
+                                        "colno": 39,
+                                    }
+                                ]
+                            },
+                            "raw_stacktrace": {
+                                "frames": [
+                                    {
+                                        "abs_path": "https://example.com/application.js",
+                                        "lineno": 1,
+                                        "colno": 39,
+                                    }
+                                ]
+                            },
+                        }
+                    ]
+                },
+            },
+            project_id=self.project.id,
+        )
+        resp = self.get_success_response(
+            self.organization.slug,
+            self.project.slug,
+            event.event_id,
+            frame_idx=0,
+            exception_idx=0,
+        )
+
+        assert resp.data["errors"] == []
```

Comment:
since the SDK version is `7.1.0` (lower than `JS_VERSION_FOR_DEBUG_ID`), shouldn't the response contain an error with `showMigrationGuide` equal `False`?

Reply:
```
>>> "7.1.0" < "7.44.0"
True
```

Reply:
never mind...just saw that in case of `raw_stack_trace`, we return something else https://github.com/getsentry/sentry/blob/76492586b8fdff0876910161b231537444a935ad/src/sentry/api/endpoints/source_map_debug.py#L88